online-shop/src/app/app.component.html

```diff
@@ -0,0 +1,7 @@
+<img src = "https://tse1.mm.bing.net/th/id/OIP.RCalqXxO0ZZxxWBS2QcMqQHaEo?w=256&h=180&c=7&r=0&o=5&dpr=1.25&pid=1.7">
```

Comment:
**suggestion:** you should keep images as already downloaded assets, try not to use direct links unless its from a clear trusted source